integration-tests/steps/stack_groups.py

```diff
@@ -266,6 +266,12 @@ def step_impl(context, first_stack, second_stack):
     assert creation_times[stacks[0]] < creation_times[stacks[1]]
 
 
+@then('stack_group drift status is "{desired_status0}" and "{desired_status1}"')
+def step_impl(context, desired_status0, desired_status1):
+    assert context.output[0]["StackDriftStatus"] == desired_status0
+    assert context.output[1]["StackDriftStatus"] == desired_status1
```

Comment:
Did you mean to remove this step, since you created a similar one in the drift steps file?

Reply:
yep